provider/tests/go.mod

```diff
@@ -5,15 +5,14 @@ go 1.21
 replace (
 	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20230710100801-03a71d0fca3d
 	github.com/hashicorp/terraform-provider-aws => ../../upstream
-	github.com/hashicorp/terraform-provider-aws/shim => ../shim
 	github.com/hashicorp/vault => github.com/hashicorp/vault v1.2.0
 	github.com/pulumi/pulumi-aws/provider/v6 => ../
 )
 
 require (
 	github.com/pulumi/pulumi-aws/provider/v6 v6.0.0-00010101000000-000000000000
-	github.com/pulumi/pulumi-terraform-bridge/pf v0.16.0
-	github.com/pulumi/pulumi-terraform-bridge/testing v0.0.1
+	github.com/pulumi/pulumi-terraform-bridge/pf e1ac6a1023a9840cd59e1e68ba7ab87174c5939b
```

Comment:
This doesn't look like it received a `go mod tidy`? Or does Go allow hash refs now?

Reply:
I don't believe that these tests are run, so my assumption is it missed the go mod tidy.